tests/test_routes.py

```diff
@@ -57,8 +56,9 @@ def setUpClass(cls):
     app.config["DEBUG"] = False
     app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
     app.logger.setLevel(logging.CRITICAL)
-    Product.init_db(app)
-    Wishlist.init_db(app)
+    with app.app_context():
+      db.drop_all()
+      db.create_all()
```

Comment:
Should this include `Product.init_db(app)` and `Wishlist.init_db(app)` like the other two?